Source/Scene/ShadowMap.js

```diff
@@ -137,11 +137,11 @@ define([
      */
     function ShadowMap(options) {
         options = defaultValue(options, defaultValue.EMPTY_OBJECT);
-        var scene = options.scene;
+        var context = options.context;
```

Comment:
Since this object is not meant to be constructed directly, it is OK for it to take `context`, but let's remove it from the above doc:
```javascript
* @param {Context} options.context The context in which to create the shadow map.
```